server/server.py

```diff
@@ -24,34 +24,37 @@ def handle(self):
 		try:
 			msg = self.RSAcipher.decrypt(data).decode().split(" ", 1)
 
-			if (msg[0] == '3317BLT5'):
+			if (msg[0] != '3317BLT5'):
+				raise BaseException("Invalid magic number")
+
 			with open("/var/log/locationLog", "a") as logFile:
 				date = time.localtime()
 				logFile.write("{0}  {1:02} {2:02}:{3:02}:{4:02}  {6} Checking in at {5}\n".format(months[date.tm_mon - 1], date.tm_mday, date.tm_hour, date.tm_min, date.tm_sec, self.client_address[0], msg[1].strip()))
 				print("[+] IP Logged to file")
+			
+		except BaseException as e:
+			print("[-] Failure: {}".format(e))
```

Comment:
right now everything is piped to stdout. This works well with systemd because you can view the past messages, to some extent, with a command like `systemctl status locationLog.service`. We can definitely look into making a stderr file or some other better logging system, but that will need to be on a different issue.